test/runtests.jl

```diff
@@ -35,17 +35,54 @@ datadir = joinpath(@__DIR__,"data")
   end
 
   @testset "Legacy" begin
-    fname = joinpath(datadir,"legacy.gslib")
-
-    sdata = GslibIO.load_legacy(fname, (2,2,2))
-    @test size(domain(sdata)) == (2,2,2)
-    @test origin(domain(sdata)) == [0.,0.,0.]
-    @test spacing(domain(sdata)) == [1.,1.,1.]
-    por = sdata[:Porosity]
-    lit = sdata[:Lithology]
-    sat = sdata[Symbol("Water Saturation")]
-    @test isequal(por, [0.1,0.2,0.3,0.4,0.5,0.6,0.7,0.8])
-    @test isequal(lit, [1,2,3,4,5,6,7,8])
-    @test isequal(sat, [0.8,0.7,0.8,0.7,0.8,0.7,0.8,NaN])
+    fname = joinpath(datadir, "legacy_grid.gslib")
+
+    sdata_grid = GslibIO.load_legacy(fname, (2, 2, 2))
+    @test size(domain(sdata_grid)) == (2, 2, 2)
+    @test origin(domain(sdata_grid)) == [0., 0., 0.]
+    @test spacing(domain(sdata_grid)) == [1., 1., 1.]
+    por = sdata_grid[:Porosity]
+    lit = sdata_grid[:Lithology]
+    sat = sdata_grid[Symbol("Water Saturation")]
+    @test isequal(por, [0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8])
+    @test isequal(lit, [1, 2, 3, 4, 5, 6, 7, 8])
+    @test isequal(sat, [0.8, 0.7, 0.8, 0.7, 0.8, 0.7, 0.8, NaN])
```

Comment:
Well, I though that the space in function call is the convention, therefore I assumed that it was the perfect time to make the changes. If we don't, do you want that the new code does not used a white space to avoid mixing formats?

Reply:
You tend to mix up changes in a pull request that have different nature. It is fine to do so after other members have reviewed your work, but showing a big diff with whitespace changes isn't very helpful for reviewers who want to make sure that the PR does not introduce bugs.

So if you spot these inconsistencies, we will be happy to fix them. But this PR is about a major change to the code to support point sets. It is already a lot of changes to review. Can you see why reviewers would be skeptical about such changes here? You need to put yourself in the side of the reviewer always. Imagine that you have a project that is out there for a while and someone submits a PR with a lot of unrelated changes. Besides the difficulty to review the work, we also mix up commits of different nature and this is hard to track in the future or to revert in case a bug is found.

Reply:
You haven't answer my question. Should I use or not white space in the new code given the current code does not use white space?

Reply:
In the new code you can always be consistent with the conventions adopted. The issue is in the old code in the tests that were changed. Strictly speaking we can always hide whitespace changes in git diffs, but my comment about reversing commits with bugs still holds. If someone wants to revert your commit, they will also revert the whitespace changes unintentionally. We want to avoid such things.

So in summary: new code can be written according to the convention. wrong conventions in old code can be fixed with separate commits, preferably in separate PRs.